event-adapter/.gitignore

```diff
@@ -0,0 +1,5 @@
+*.swp
+*~
+sample-main
```

Comment:
Also, please remove file build/adapter from the repo. I think it got included at some point by mistake when .gitignored was deleted.